shared/macros-ansible.jinja

```diff
@@ -173,3 +173,14 @@
 {{%- macro ansible_auditd_set(msg='', parameter='', value='') %}}
 {{{ ansible_set_config_file(msg, "/etc/audit/auditd.conf", parameter=parameter, value=value, create="yes", separator=" = ", separator_regex="\s*=\s*") }}}
 {{%- endmacro %}}
+
+{{#
+  High level macro to set a parameter in /etc/systemd/coredump.conf.
+  Parameters:
+  - msg: the name for the Ansible task
+  - parameter: parameter to be set in the configuration file
+  - value: value of the parameter
+#}}
+{{%- macro ansible_coredump_config_set(msg='', parameter='', value='') %}}
+{{{ ansible_set_config_file(msg, "/etc/systemd/coredump.conf", parameter=parameter, value=value, create="yes", separator="=", separator_regex="\s*=\s*") }}}
```

Comment:
Unfortunately this macro is not able to handle `ini` configuration file. Ansible provides a module to handle ini files:
https://docs.ansible.com/ansible/latest/modules/ini_file_module.html

In the project it is being used in some cases:
https://github.com/ComplianceAsCode/content/blob/3ea35b6431430e81fdf9303fc58598454999b3ca/linux_os/guide/services/sssd/sssd_offline_cred_expiration/ansible/shared.yml#L13-L20

We would need to create a new macro wrapping this `ini_file` from ansible. For now it is ok to leave like that.